src/util/util.js

```diff
@@ -16,3 +16,7 @@ export const noop = () => {};
 export const noargs = (f) => () => f();
 
 export const sumUnderThreshold = (list, threshold) => list.reduce((acc, i) => acc + Math.min(i, threshold), 0);
+
+export const getCsrf = () => decodeURIComponent(document.cookie.split(';')
```

Comment:
When I `console.log(document.cookie)`, I see that the separator is `'; '` with a space after the semicolon. MDN [suggests](https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie#read_all_cookies_accessible_from_this_location) that the amount of white space may vary by browser:

> Note that each _key_ and _value_ may be surrounded by whitespace (space and tab characters): in fact, RFC 6265 mandates a single space after each semicolon, but some user agents may not abide by this.

Maybe we could split by `/;[ \t]*/`?